package.json

```diff
@@ -106,20 +154,6 @@
     "webpack-merge": "^0.14.1",
     "webpack-validator": "^2.2.9"
   },
-  "dependencies": {
```

Comment:
@zeevl I'm used and agreed [this ESLint rule](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/imports.js#L69) (Although it's disabled here), it's the reason for why I distinguish main dependencies and development tools. (Of course, exclude dev packages in `development` node env)

> FWIU anything that gets packaged with an app goes in ./app/package.json; everything else belongs in the root ./package.json

I prefer just put webpack externals to dependencies of `./app/package.json`, we can focus on __native__ modules, sometimes I can only add deeper dependencies to reduce bundle size, so I think keep dependencies of `./package.json` not bad, 
it is not related to production build.